### PR TITLE
control: set the highest priority for control service.

### PR DESCRIPTION
**Pull request by AliceInHunterland**
Closes: #2585.

### CODE REVIEW

File: cmd/neofs-node/control.go [hunk 1]
```diff
@@ -64,11 +64,11 @@ func initControlService(c *cfg) {
 
 	control.RegisterControlServiceServer(c.cfgControlService.server, ctlSvc)
 
-	c.workers = append(c.workers, newWorkerFromFunc(func(ctx context.Context) {
-		runAndLog(c, "control", false, func(c *cfg) {
+	go func() {
```
Comment (roman-khimov) [inline cmd/neofs-node/control.go:67]: Why async? It can start at any point in time then.

**Review by roman-khimov — COMMENTED** [review]
IR?

**Review by carpawell — COMMENTED** [review]
The commit starts with "conrtol".

**Comment by roman-khimov** [comment]
Needs to be rebased.

File: CHANGELOG.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ Changelog for NeoFS Node
 - IR uses internal LOCODE DB from Go package (#2610)
 - Contract group for the committee is no longer registered/used in the Sidechain auto-deployment (#2642)
 - The priority of metrics service is increased (#2428)
+- The priority of running control service is increased (#2585)
```
Comment (cthulhu-rider) [inline CHANGELOG.md:29]: if `set the highest priority`, then `maximized`

File: pkg/innerring/innerring.go [hunk 3]
```diff
@@ -519,6 +519,52 @@ func New(ctx context.Context, log *zap.Logger, cfg *viper.Viper, errChan chan<-
...
+		p.SetHealthChecker(server)
+
+		controlSvc := controlsrv.New(p,
+			controlsrv.WithAllowedKeys(authKeys),
+		)
+
+		grpcControlSrv := grpc.NewServer()
+		control.RegisterControlServiceServer(grpcControlSrv, controlSvc)
+
+		lis, err := net.Listen("tcp", controlSvcEndpoint)
```
Comment (cthulhu-rider) [inline pkg/innerring/innerring.go:551]: `err` is lost

btw best to start listening immediately after parsing the keys in order to respond to an error as early as possible, and only if everything is ok - init gRPC server

**Review by cthulhu-rider — CHANGES_REQUESTED** [review]
commit header ends with a period. It is unclear from the text that priority has been maxed for both SN and IR

**Review by carpawell — APPROVED** [review]

**Pull request merged**